packages/issuer/lib/VcIssuer.ts

```diff
@@ -57,13 +56,17 @@ export class VcIssuer {
    * @param issueCredentialRequest a credential issuance request
    * @param jwtVerifyCallback OPTIONAL. if provided will use this callback instead what is configured in the VcIssuer
    * @param issuerCallback OPTIONAL. if provided will use this callback instead what is configured in the VcIssuer
+   * @param issuerState the state of the issuer
    */
   public async issueCredentialFromIssueRequest(
     issueCredentialRequest: CredentialRequest,
+    issuerState: string,
```

Comment:
issuerState is something which is version specific. It was called opState before. Please double check these types of things